android/src/main/java/com/twiliovoicereactnative/CallMessageListenerProxy.java

```diff
@@ -0,0 +1,74 @@
+package com.twiliovoicereactnative;
+
+import static com.twiliovoicereactnative.CommonConstants.CallEventMessageFailure;
+import static com.twiliovoicereactnative.CommonConstants.CallEventMessageReceived;
+import static com.twiliovoicereactnative.CommonConstants.CallEventMessageSent;
+import static com.twiliovoicereactnative.CommonConstants.CallMessageSID;
+import static com.twiliovoicereactnative.CommonConstants.ScopeCall;
+import static com.twiliovoicereactnative.CommonConstants.VoiceErrorKeyError;
+import static com.twiliovoicereactnative.CommonConstants.VoiceEventType;
+import static com.twiliovoicereactnative.Constants.JS_EVENT_KEY_CALL_MESSAGE_INFO;
+import static com.twiliovoicereactnative.JSEventEmitter.constructJSMap;
+import static com.twiliovoicereactnative.ReactNativeArgumentsSerializer.serializeCallMessage;
+import static com.twiliovoicereactnative.ReactNativeArgumentsSerializer.serializeVoiceException;
+import static com.twiliovoicereactnative.VoiceApplicationProxy.getJSEventEmitter;
+
+import android.util.Pair;
+
+import androidx.annotation.NonNull;
+
+import com.facebook.react.bridge.WritableMap;
+import com.twilio.voice.Call;
+import com.twilio.voice.CallMessage;
+import com.twilio.voice.VoiceException;
+
+public class CallMessageListenerProxy implements Call.CallMessageListener {
```

Comment:
There's two more entry points that we'll need to instantiate a `CallMessageListener`, `voice.connect` for outgoing calls, and `callInvite.accept` for incoming calls.